README.md

```diff
@@ -1,3 +1,29 @@
-# Requirements
-Boost 1.76.0
-OpenSSL
\ No newline at end of file
+## Requirements
+Boost 1.76.0  
+OpenSSL  
+## Compile project and run test under ubuntu 20.04.
+```
+sudo apt-get update 
+sudo apt install g++
+sudo apt-get install -y libssl-dev
+
+# compile gtest
+# https://zwarrior.medium.com/install-google-test-framework-gtest-on-ubuntu-20-04-368eb6951b12
+sudo apt-get install -y libgtest-dev
+cd /usr/src/gtest
+sudo cmake CMakeLists.txt
```

Comment:
[Optional] It would preferred to use the system's gtest.

Reply:
Prefer keeping it just in case.